### PR TITLE
expose new RPC for loading UPAK2 w/ all incarnations

### PR DESCRIPTION
**Pull request by maxtaco**
- simple wrapper around UPAK loader

### CODE REVIEW

**Comment by maxtaco** [comment]
cc: @strib

File: go/protocol/keybase1/extras.go [hunk 1]
```diff
@@ -1351,6 +1351,11 @@ func (u UserPlusKeysV2AllIncarnations) FindKID(kid KID) (*UserPlusKeysV2, *Publi
 	return nil, nil
 }
 
+func (u UserPlusKeysV2AllIncarnations) HasKID(kid KID) bool {
```
Comment (mlsteele) [inline go/protocol/keybase1/extras.go]: I'd be a little surprised that HasKID includes previous incarnations. Add comment?

**Comment by maxtaco** [comment]
Thanks, I'll fix the comment and also fix broken tests in chat/ due to new method on the UPAK loader.

**Comment by maxtaco** [comment]
Merging over CI failures. URGGGGG

**Pull request merged**